content/en/serverless/installation/dotnet.md

```diff
@@ -13,69 +13,188 @@ further_reading:
   text: 'Submitting Custom Metrics from Serverless Applications'
 ---
 
-## Prerequisites
+To instrument your .NET serverless application, you must use **either** the [Datadog Lambda Extension][1] (beta) or the [Datadog Forwarder Lambda function][2].
 
-To instrument your .NET serverless application, you must install **either** the [Datadog Forwarder Lambda function][1] or (beta) the [Datadog Lambda Extension][2].
+If you use the Datadog Lambda Extension, you can use Datadog's native .NET APM tracer to instrument your function. If you use the Datadog Forwarder Lambda function, you can use Datadog's integration with AWS X-Ray tracing.
 
-## Configuration
+**Note**: The Datadog Lambda Extension only supports the `x86_64` architecture for .NET Lambda functions. If your .NET Lambda function uses the `arm64` architecture, you must use the Datadog Forwarder for instrumentation.
 
-{{< tabs >}}
-{{% tab "Extension" %}}
+## Instrumentation
+### Using the Datadog Lambda Extension
 
 <div class="alert alert-warning">
 Instrumenting .NET applications with the Datadog Lambda Extension is in beta.
 </div>
 
-Add the [Datadog Lambda Extension][1] layer for your Lambda function using the appropriate ARN for your desired processor:
+Datadog offers many different ways to enable instrumentation for your serverless applications. Choose a method below that best suits your needs.
 
-**x86**: `arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Extension:<EXTENSION_VERSION>`
+{{< tabs >}}
+{{% tab "Serverless Framework" %}}
 
-**ARM**: `arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Extension-ARM:<EXTENSION_VERSION>`
+1. Add the following layers and environment variables to each .NET Lambda function you wish to instrument:
 
-Set the environment variable `DD_API_KEY` to your [Datadog API key][2].
+    ```yml
+    your-function:
+      layers:
+        - arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Extension:{{< latest-lambda-layer-version layer="extension" >}}
+        - arn:aws:lambda:<AWS_REGION>:464622532012:layer:dd-trace-dotnet:{{< latest-lambda-layer-version layer="dd-trace-dotnet" >}}
+      environment:
+        DD_TRACE_ENABLED: true
+        DD_API_KEY: "<YOUR_DD_API_KEY>"
+        CORECLR_ENABLE_PROFILING: 1
+        CORECLR_PROFILER: "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}"
+        CORECLR_PROFILER_PATH: "/opt/datadog/Datadog.Trace.ClrProfiler.Native.so"
+        DD_DOTNET_TRACER_HOME: "/opt/datadog"
+    ```
 
+    Replace `<YOUR_DD_API_KEY>` with your Datadog API key, found on the [API Management page][1].
 
-[1]: /serverless/libraries_integrations/extension/
+2. Optionally add `service` and `env` tags with appropriate values to your function.
```

Comment:
Should we add something after this to make it clear that the reader is done and doesn't need to follow the Forwarder instructions?

Reply:
I added a line at the beginning of the Forwarder section to this effect, do you think it's not enough?